use pm2 instead of supervisord

daniel-rose opened the pull request:
- add newer php version (7.3)
- refactoring
- makefile for build and push
- add travis workflow
- ...

daniel-rose committed:
use pm2 instead of supervisord
- add newer php version (7.3)
- refactoring
- makefile for build and push
- add travis workflow
- ...

daniel-rose committed:
use pm2 instead of supervisord
- add newer php version (7.3)
- refactoring
- makefile for build and push
- add travis workflow
- ...

sourcecube commented on docker-sync.yml:
```diff
@@ -0,0 +1,13 @@
+syncs:
+  spryker-sync:
+    src: 'PATH_TO_SOURCE'
+    sync_excludes: ['EXCLUDE_1', '...', '.EXCLUDE_N']
```
Is that correct?

daniel-rose replied:
The docker-compose.yml and docker-sync.yml are starting points for beginners.

daniel-rose replied:
Maybe the file extension .dist is better for this files.

sourcecube commented on php/7.3/base/apache.Dockerfile:
```diff
@@ -0,0 +1,79 @@
+    locale-gen
+
+# composer
+RUN set -ex; \
+    \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        git \
+        zip \
+        libzip-dev; \
+    rm -rf /var/lib/apt/lists/*; \
+    \
+    docker-php-ext-configure zip --with-libzip; \
+    docker-php-ext-install zip; \
+    \
+    wget https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer -O /tmp/installer; \
+    php /tmp/installer --no-ansi --install-dir=/usr/bin --filename=composer --quiet; \
+    rm /tmp/installer; \
+    composer self-update
```
Is there a reason to add empty lines here?

daniel-rose replied:
Better to read?!

daniel-rose committed:
use pm2 instead of supervisord
- add dist ext to docker-compose.yml and docker-sync.yml
- add .env.dist for docker-compose.yml.dist
- remove log from makefile
